### ISSUE TITLE
new assumptions

### ISSUE THREAD

**Issue by certik** [sympy/sympy-paper#180]
@skirpichev reported: Shouldn't you mention some problems with
the current state of arts in assumptions?  And how you are
planning to solve this mess.

**Comment by asmeurer** [comment]
I agree that the new assumptions are a bit of a mess (or as I would call it, a "work in progress"), which is one of the reasons that I've decided to focus mainly on the old assumptions for the paper. 

All the little bits that I had mentioning the new assumptions ended up being deleted. It would be nice to have a longer section, perhaps in the supplement, discussing them, but I don't know if it will happen. I am not personally working full time on the paper anymore, so either I write it in my off time, or someone else write it.
